source/ddox/entities.d

```diff
@@ -612,6 +612,7 @@ struct Type {
 	immutable(CachedType)[] parameterTypes;
 	immutable(CachedString)[] _parameterNames;
 	immutable(Value)[] _parameterDefaultValues;
+	bool variadic, typesafeVariadic;
```

Comment:
Would be a bt nicer to use an `enum VariadicKind { none, generic, typeSafe }` here.

Reply:
Done, just used [std.traits.Variadic](https://dlang.org/library/std/traits/variadic.html).